css-view-transitions-2/Overview.bs

```diff
@@ -1079,55 +1084,63 @@ When capturing the old or new state for an element, perform the following steps
+
+	1. Let |containingGroupName| be |capturedElement|'s [=containing group name=].
+	1. If |containingGroupName| is not null, then:
 
-	1. If |capturedElement|'s [=containing group=] is not null, then:
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|containingGroupName|].
 
 		1. Let |group| be the ''::view-transition-group()'',
-			with its [=view transition =name=] set to |transitionName|.
+			whose [=view transition name=] is set to |transitionName|.
 
 		1. Let |parentGroup| be the ''::view-transition-group()'',
-			with its [=view transition name=] set to |capturedElement|'s [=containing group=].
+			whost [=view transition name=] is set to |containingGroupName|.
+
+		1. Set |capturedElement|'s [=captured element/old transform=] to the result of [=multiplying=] |capturedElement|'s [=captured element/old transform=] by the inverse matrix of |groupContainerElement|'s [=captured element/old transform=].
 
 		1. Append |group| to |parentGroup|.
 
 	Note: It is TBD how this is resolved when the old and new groups mismatch. See <a href="https://github.com/w3c/csswg-drafts/issues/10631">Issue 10631</a>.
 </div>
 
+### Adjust the group's 'transform' to be relative to its containing ''::view-transition-group()'' ### {#vt-group-transform-adjust}
 <div algorithm="additional pseudo-element style update steps (group)">
-When updating the style of the transition pseudo-element for a [=captured element=] |capturedElement|, given a |transitionName| and a {{ViewTransition}} |transition|:
+When [[update pseudo-element styles|updating the style of the transition pseudo-element]], perform the following steps before setting the [=captured element/group styles rule=], given a [=captured element=] |capturedElement|, given a |transform| and a {{ViewTransition}} |transition|:
 
-	1. If |capturedElement|'s [=containing group=] is not null, then:
+	1. Set |capturedElement|'s [=transform from snapshot containing block=] to |transform|.
 
-		1. Let |group| be the ''::view-transition-group()'',
-			with its [=view transition name=] set to |transitionName|.
-
-		1. Let |containingGroupName| be |transition|'s [=ViewTransition/named elements=][|capturedElement|'s [=containing group=].
+	1. If |capturedElement|'s [=containing group name=] is not null, then:
 
-		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|containingGroupName|].
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|capturedElement|'s [=containing group name=]].
 
-		1. Set |group|'s 'transform' to the transform that would map |capturedElement|’s [=new element=] [=border box=] from |groupContainerElement|'s [=border box=] to its current visual position.
+		1. [=Multiply=] |transform| by the inverse matrix of |groupContainerElement|'s [=transform from snapshot containing block=].
```

Comment:
This looks like a bug for the exit transition case. Since |transform| passed to this algorithm would be the "old transform" here: https://drafts.csswg.org/css-view-transitions-1/#style-transition-pseudo-elements-algorithm:~:text=Set%20transform%20to%20capturedElement%E2%80%99s%20old%20transform.

And we already multiplied the "old transform" with an inverse of its ancestor's transform in "setting up the transition pseudo-element" above?

Reply:
Fixed, not multiplying in the previous phase.